site2/website-next/versioned_docs/version-2.6.3/administration-zk-bk.md

```diff
@@ -205,6 +210,9 @@ ledgerDirectories=data/bookkeeper/ledgers
 # Point to local ZK quorum
 zkServers=zk1.example.com:2181,zk2.example.com:2181,zk3.example.com:2181
 
+#This parameter best be set, otherwise，the Bookkeeper can't start normally in special environment
```

Comment:
Curious what the special environment is, is it possible to describe the special environment in a few sentences ?  Also if not set will there be a default value for this `advertisedAddress` ?

Reply:
As far as I know, Huawei Cloud has this problem, and other special scenarios are unknown.In the Huawei Cloud environment, if it is not configured, bookkeeper cannot be started normally.

Reply:
Cool ~ Maybe We can say "...the Bookkeeper can't start normally in Huawei Cloud for now"? What do you think ~